content/kotlin/kotlin.md

```diff
@@ -0,0 +1,21 @@
+---
+Title: 'Kotlin'
+Description: 'Kotlin is a programming language used for general purposes including front-end, back-end, and mobile development.'
+Codecademy Hub Page: 'https://www.codecademy.com/catalog/language/kotlin'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+[**Kotlin**](https://www.codecademy.com/catalog/language/kotlin) is a programming language used for general purposes including [front-end](https://www.codecademy.com/resources/docs/general/front-end), [back-end](https://www.codecademy.com/resources/docs/general/back-end), and mobile development. Since Kotlin code is fully compatible with [Java](https://www.codecademy.com/resources/docs/java), its syntax may seem familiar to those with proficiency in Java.
```

Comment:
```suggestion
[**Kotlin**](https://www.codecademy.com/catalog/language/kotlin) is a programming language used for general purposes including [front-end](https://www.codecademy.com/resources/docs/general/front-end), [back-end](https://www.codecademy.com/resources/docs/general/back-end), and mobile development. Since Kotlin is fully compatible with [Java](https://www.codecademy.com/resources/docs/java), its syntax may seem familiar to those with proficiency in Java.
```